exploring-national-parks/src/HomePage/Components/Buttons.jsx

```diff
@@ -15,12 +15,33 @@ const Buttons = () => {
         window.location.href = "ParkSearch"
     }
 
+    async function randPark() {
+      try {
+        const url = `https://developer.nps.gov/api/v1/parks?api_key=Y7kFnm6SP5SMQhkTvwUSgyjge9buj4DbjrkuV2S0&limit=471`;
+        const response = await fetch(url);
```

Comment:
We already fetch these elsewhere like on the homepage; we can just reuse the data we already have. Use the other PR as an example.